lib/sms.rb

```diff
@@ -0,0 +1,24 @@
+require 'twilio-ruby'
+
+class Sms
+
+  def initialize
+    account_sid = 'AC5f0916482a7dd3822a118d4b27a4e49b'
+    auth_token = 'a58d36544e1daf403e1d3448c820cfd1'
+    @client = Twilio::REST::Client.new account_sid, auth_token
+  end
+
+  def message
+    message = "Thank you! Your order:#{order_number} was placed"\
+              " and will be delivered before"\
+              " #{Time.now.hour + 1}:#{Time.now.min}"
+  end
+
+  def send(order_number)    
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `order_number`. If it's necessary, use `_` or `_order_number` as an argument name to indicate that it won't be used. You can also write as `send(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)